express/app/middlewares/userFieldsValidator.js

```diff
@@ -0,0 +1,11 @@
+const errors = require('./errors');
+
+const requiredFields = ['firstName', 'lastName', 'email', 'password'];
+
+exports.handle = (req, res, next) => {
+  const missingFields = requiredFields.filter(field => !req.body[field]);
+  if (missingFields.length > 0) {
+    errors.handle(`${missingFields} are required`, req, res, next);
+  }
+  next();
```

Comment:
this is ok to check the fields, but you don't validate if the firstName or lastName are string.